mappings/net/minecraft/world/chunk/Chunk.mapping

```diff
@@ -3,39 +3,27 @@ CLASS net/minecraft/class_399 net/minecraft/world/chunk/Chunk
 	FIELD field_1563 chunkZ I
 	FIELD field_1564 LOGGER Lorg/apache/logging/log4j/Logger;
 	FIELD field_1567 minRainOrSnowHeight [I
-		COMMENT Stores the min y-value at which rain or snow can fall for a given (x, z) position
-	FIELD field_1568 updateLight [Z
-		COMMENT the (x, z) positions that need their skylight updated.
+	FIELD field_1568 needsLightUpdate [Z
 	FIELD field_1569 loaded Z
-		COMMENT This value is true if the chunk is currently loaded into the world
```

Comment:
Was the removal of all the javadocs in this file intentional or enigma doing a funny? They seem pretty reasonable overall.

Reply:
Oh yikes, I definitely didn't intend to remove all that. I was editing the javadocs here but idk what went wrong.